datafusion/optimizer/src/filter_null_join_keys.rs

```diff
@@ -74,13 +74,15 @@ impl OptimizerRule for FilterNullJoinKeys {
                     join.left = Arc::new(LogicalPlan::Filter(Filter::try_new(
                         predicate,
                         join.left.clone(),
+                        20,
```

Comment:
This seems a case where the optimizer doesn't really optimize an existing filter, but add a filter, so I think we can only default it